tests/python/unittest/test_micro_model_library_format.py

```diff
@@ -208,7 +208,12 @@ def @main(%a : Tensor[(1, 2), uint8], %b : Tensor[(1, 2), float32], %c : Tensor[
                 {
                     "constants_size_bytes": json_constants_size_bytes,
                     "device": 1,
+                    "inputs": {
+                        "a": 2,
+                        "b": 8,
+                    },
                     "io_size_bytes": 18,
+                    "outputs": {"0": 8},
```

Comment:
This should match with the name of the output in the generated C (`output` for single, `outputX` for multiple, or the actual name if we can get to it)

```suggestion
                    "outputs": {"output": 8},
```